DeepSpeech.py

```diff
@@ -960,6 +1027,54 @@ def main(_):
         tfv1.reset_default_graph()
         do_single_file_inference(FLAGS.one_shot_infer)
 
+    if FLAGS.finetune_lm_csv_files:
```

Comment:
Please move this into a separate file, similar to how `evaluate.py` is structured.